libraries/chain/include/eosio/chain/finality/finalizer.hpp

```diff
@@ -70,21 +71,34 @@ namespace eosio::chain {
    // ----------------------------------------------------------------------------------------
    struct my_finalizers_t {
    public:
-      static constexpr uint64_t current_safety_file_version = 0;
+      ///
+      /// Version 0: Spring 1.0.0 RC2 - File has fixed packed sizes with inactive safety info written to the end
+      ///                               of the file. Consists of [finalizer public_key, FSI]..
+      /// Version 1: Spring 1.0.0 RC3 - Variable length FSI with inactive safety info written at the beginning of
+      ///                               the file. Uses crc32 checksum to verify data on read.
+      ///
+      static constexpr uint64_t safety_file_version_0       = 0;
+      static constexpr uint64_t safety_file_version_1       = 1;
```

Comment:
I don't think these are very useful, but OK.

Reply:
Maybe they represent all valid options? 🤷  Not sure, but seems better than just 0 and 1 hard coded.

Reply:
I feel since `version_0` will always be `0` and `version_1` will always be `1`, hardcoded is just as good and one less indirection, but personal preference so no need to change.